---
layout: default
title: Amalia Mart√≠nez Segura
---

<div class="cv_section">
  <h1>Summary</h1>
  <p> Currently I am a researcher at the London Institute of Medical Sciences.
    Although I am a molecular biologist by training, I have been interpreting and visualizing the genome-wide datasets I produce experimentally using R since I started my PhD.
    My research involves the use of statistics to interpret complex biological datasets to produce testable hypotheses, in close collaboration with other research groups.
    I am interested in applying and developing my programming and data analysis skills. </p>
</div>

<div class='cv_section'>
  <h1>Omics analysis</h1>
  <ul>
    <li>Experience generating NGS data using Illumina protocols and mass-spectrometry proteomics
    <li>Use of next generation sequencing quality control (FastQC) and visualisation tools (igv)
    <li>Familiar with alignment tools (TopHat2, Bowtie2) and other software necessary for the processing of raw reads (samtools, bedtools, HTseq)
    <li>Knowledge of biological databases (UCSC, Ensembl, KEGG, GEO), the Bioconductor package ecosystem and Ingenuity Pathway Analysis software (IPA)
    <li>Understanding of the computational pipeline to analyse mass-spectrometry data (Mascot, Progenesis)
  </ul>
  <h1>Data analysis</h1>
  <ul>
    <li>Identified transcripts involved in the activation of stem cells in the mouse brain using R (DESeq2), combined with the output from Ingenuity Pathway Analysis
    <li>Used R (ggplot2) to produce rich visualisations (like these ones) of omics datasets
    <li>Determined transcripts and proteins correlated with growth rate when fission yeast is exposed to different environments
    <li>Able to mine different kinds of biological data (ChIP-seq, RNA-seq, proteomics) and integrate them in order to explore new hypotheses or support conclusions from other analyses
  </ul>
  <h1>Programming</h1>
  <ul>
    <li>Used R to infer differentially expressed transcripts and proteins from time-course proteomics and transcriptomics dataset representing the majority of molecules inside fission yeast cells
    <li>Parsed text files from a Matlab software (FISH-quant) using Python to create clean datasets that could be visualised easily
    <li>Knowledge of Linux and HPC environments, where I implemented a pipeline in shell that integrates all tools to process raw next generation sequencing data into a format amenable for statistical analysis
    <li>Use of version control software (Git) together with an R IDE (R Studio)
  </ul>
  <h1>Communication</h1>
  <ul>
    <li>Carried out productive collaborations with groups in other fields such as mathematics and cell biology
    <li>Presented my research at two international conferences (qBio in Nashville, and the Pombe Meeting in Barcelona) with more than 200 attendees
    <li>Gave talks about synthetic biology at two high schools in my home town as part of an initiative to increase awareness of the subject
    <li>Supervised 3 MSc. and 2 BSc. students during their stay in the laboratory, helping them manage their own projects and teaching how to carry out experiments
  </ul>
</div>

<div class="cv_section">
  <h1>Career History</h1>
  <h2>Postdoctoral Researcher and PhD. Student</h2>
  <ul>
    <li>
    
</div>

<div class="cv_section">
  <h1>Education</h1>
</div>

<div class="cv_section">
  <h1>Extracurricular activities</h1>
</div>
